hmri_quiqi_check.m

```diff
@@ -129,10 +129,8 @@ function plotResFit(Y,yfit,Rsq,Nbins,SavePath)
 % function of the MDI
 
 figure
-[N1, c] = hist3(cat(2,Y,yfit),{linspace(min(min(Y),min(yfit)),max(max(Y),max(yfit)),Nbins) linspace(min(min(Y),min(yfit)),max(max(Y),max(yfit)),Nbins) },'CdataMode','auto');
-h=pcolor(c{1}, c{2}, N1');set(h, 'EdgeColor', 'none');
-colorbar;view(2)
```

Comment:
Perhaps you could use [`histogram2`](https://www.mathworks.com/help/matlab/ref/matlab.graphics.chart.primitive.histogram2.html) rather than replacing the histogram with a line plot?

Reply:
Thanks for the suggestion, we believe that "plot" is sufficient so let's keep "plot" if you don't see any issue with that.

Reply:
That should be fine, but I guess you will need to update the examples [here](https://github.com/hMRI-group/hMRI-toolbox/wiki/QUIQI).